<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Doki-Doki-Just-Monika</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1278</width>
      <height>719</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rs_ddjm.qrc">:/Resource/bg.JPG</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_gif">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>320</width>
      <height>188</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 230, 244);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_word_Sayori">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(133, 100, 168, 128);
color:white;
font: 16px &quot;Bauhaus 93&quot;;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}

QPushButton:hover{
background-color: rgb(255, 189, 225);
	color: rgb(200, 142, 242);
font: 16px;
border:2px solid gray;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
QPushButton:pressed{
background-color: rgb(255, 228, 245);
color:white;
font: 16px;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
</string>
    </property>
    <property name="text">
     <string>S Words</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_word_Yuri">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(133, 100, 168, 128);
color:white;
font: 16px &quot;Bauhaus 93&quot;;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}

QPushButton:hover{
background-color: rgb(255, 189, 225);
	color: rgb(200, 142, 242);
font: 16px;
border:2px solid gray;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
QPushButton:pressed{
background-color: rgb(255, 228, 245);
color:white;
font: 16px;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
</string>
    </property>
    <property name="text">
     <string>Y Words</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_word_Natsuki">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(133, 100, 168, 128);
color:white;
font: 16px &quot;Bauhaus 93&quot;;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}

QPushButton:hover{
background-color: rgb(255, 189, 225);
	color: rgb(200, 142, 242);
font: 16px;
border:2px solid gray;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
QPushButton:pressed{
background-color: rgb(255, 228, 245);
color:white;
font: 16px;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
</string>
    </property>
    <property name="text">
     <string>N Words</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_word_Monika">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(133, 100, 168, 128);
color:white;
font: 16px &quot;Bauhaus 93&quot;;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}

QPushButton:hover{
background-color: rgb(255, 189, 225);
	color: rgb(200, 142, 242);
font: 16px;
border:2px solid gray;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
QPushButton:pressed{
background-color: rgb(255, 228, 245);
color:white;
font: 16px;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
</string>
    </property>
    <property name="text">
     <string>M Words</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_find_bg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>320</width>
      <height>260</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(254, 190, 225, 180);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_findword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(113, 37, 113, 128);
border-color: rgb(255, 255, 255);
color: rgb(254, 230, 244);
font: 16px &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 142, 245);
font: 16px &quot;Bauhaus 93&quot;;</string>
    </property>
    <property name="text">
     <string>Search keyword here!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_find_word">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(133, 100, 168, 128);
color:white;
font: 16px &quot;Bauhaus 93&quot;;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}

QPushButton:hover{
background-color: rgb(255, 189, 225);
	color: rgb(200, 142, 242);
font: 16px;
border:2px solid gray;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
QPushButton:pressed{
background-color: rgb(255, 228, 245);
color:white;
font: 16px;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
</string>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>190</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(113, 37, 113, 128);
border-color: rgb(255, 255, 255);
color: rgb(254, 230, 244);
font: 16px &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_icon_girl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>100</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rs_ddjm.qrc">:/Resource/none-icon-mini.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_cr">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>320</width>
      <height>260</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(113, 37, 113, 128);
border-color: rgb(255, 255, 255);
color: rgb(254, 230, 244);
font: 16px &quot;Arial&quot;;</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="plainText">
     <string>DISCLAIMER

This is a free and open-source doujin work, and the copyright of all resources used belongs to DDLC (Doki Doki Literature Club). If you encounter payment requests, please check if you have obtained tampered works.
If the work infringes on your intellectual property rights, please contact the author for modification support.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_MAS">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>90</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 142, 245);
font: 16px &quot;Bauhaus 93&quot;;</string>
    </property>
    <property name="text">
     <string>MAS Support</string>
    </property>
   </widget>
   <widget class="MASSForm" name="widget_mas" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_find_bg_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>80</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(254, 190, 225, 180);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_get_mas_cn">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>40</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(133, 100, 168, 128);
color:white;
font: 16px &quot;Bauhaus 93&quot;;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}

QPushButton:hover{
background-color: rgb(255, 189, 225);
	color: rgb(200, 142, 242);
font: 16px;
border:2px solid gray;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
QPushButton:pressed{
background-color: rgb(255, 228, 245);
color:white;
font: 16px;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
</string>
    </property>
    <property name="text">
     <string>Get MAS CN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_get_mas">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>40</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgba(133, 100, 168, 128);
color:white;
font: 16px &quot;Bauhaus 93&quot;;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}

QPushButton:hover{
background-color: rgb(255, 189, 225);
	color: rgb(200, 142, 242);
font: 16px;
border:2px solid gray;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
QPushButton:pressed{
background-color: rgb(255, 228, 245);
color:white;
font: 16px;
border:2px solid white;
border-top-left-radius:4px;
border-top-right-radius:4px;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px
}
</string>
    </property>
    <property name="text">
     <string>Get MAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 142, 245);
font: 16px &quot;Bauhaus 93&quot;;</string>
    </property>
    <property name="text">
     <string>Start Support</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_word_Sayori</zorder>
   <zorder>pushButton_word_Yuri</zorder>
   <zorder>pushButton_word_Natsuki</zorder>
   <zorder>pushButton_word_Monika</zorder>
   <zorder>label_gif</zorder>
   <zorder>label_find_bg</zorder>
   <zorder>lineEdit_findword</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_find_word</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>label_icon_girl</zorder>
   <zorder>plainTextEdit_cr</zorder>
   <zorder>widget_mas</zorder>
   <zorder>label_find_bg_2</zorder>
   <zorder>pushButton_get_mas_cn</zorder>
   <zorder>pushButton_get_mas</zorder>
   <zorder>checkBox_MAS</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MASSForm</class>
   <extends>QWidget</extends>
   <header location="global">MASSForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rs_ddjm.qrc"/>
 </resources>
 <connections/>
</ui>
